addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -1,204 +1,208 @@
+                <DataDDT t-esc="format_date(record.l10n_it_ddt_id.date)"/>
+            </DatiDDT>
+        </DatiGenerali>
+        <DatiBeniServizi>
+            <t t-set="line_counter" t-value="0"/>
+            <t t-foreach="record.invoice_line_ids.filtered(lambda l: not l.display_type)" t-as="line">
+                <t t-set="line_counter" t-value="line_counter + 1"/>
+                <t t-set="taxes" t-value="line.tax_ids.compute_all(line.price_unit)"/>
+                <t t-call="l10n_it_edi.account_invoice_line_it_FatturaPA"/>
+            </t>
+            <t t-foreach="tax_details['tax_details']" t-as="tax_name">
+                <t t-set="tax_dict" t-value="tax_details['tax_details'][tax_name]"/>
+                <t t-set="tax" t-value="tax_dict['tax']"/>
+                <t t-set="has_exoneration" t-value="tax.l10n_it_has_exoneration"/>
+                <t t-set="kind_exoneration" t-value="tax.l10n_it_kind_exoneration"/>
+                <DatiRiepilogo>
+                    <AliquotaIVA t-esc="format_numbers(tax.amount)"/>
+                    <Arrotondamento t-if="tax_dict.get('rounding')" t-esc="format_numbers(tax_dict['rounding'])"/>
+                    <ImponibileImporto t-esc="format_monetary(abs(tax_dict['base_amount']), currency)"/>
+                    <Imposta t-esc="format_monetary(abs(tax_dict['tax_amount']), currency)"/>
+                    <EsigibilitaIVA t-if="not has_exoneration or kind_exoneration == 'N6'" t-esc="tax.l10n_it_vat_due_date"/>
+                    <Natura t-if="has_exoneration" t-esc="kind_exoneration"/>
+                    <RiferimentoNormativo t-if="has_exoneration" t-esc="format_alphanumeric(tax.l10n_it_law_reference[:100])"/>
+                </DatiRiepilogo>
+            </t>
+        </DatiBeniServizi>
+        <DatiPagamento t-if="partner_bank and record.move_type != 'out_refund'">
+            <t t-set="payments" t-value="record.line_ids.filtered(lambda line: line.account_id.user_type_id.type in ('receivable', 'payable'))"/>
+            <CondizioniPagamento><t t-if="len(payments) == 1">TP02</t><t t-else="">TP01</t></CondizioniPagamento>
+            <t t-foreach="payments" t-as="payment">
+                <DettaglioPagamento>
+                    <ModalitaPagamento>MP05</ModalitaPagamento>
+                    <DataScadenzaPagamento t-esc="format_date(payment.date_maturity)"/>
+                    <ImportoPagamento t-esc="format_monetary(abs(payment.price_total), currency)"/>
+                    <IstitutoFinanziario t-if="partner_bank.bank_id" t-esc="format_alphanumeric(partner_bank.bank_id.name[:80])"/>
+                    <IBAN t-if="partner_bank.acc_type == 'iban'" t-esc="partner_bank.sanitized_acc_number"/>
+                    <BIC t-if="partner_bank.acc_type == 'bank' and partner_bank.bank_id.bic" t-esc="partner_bank.bank_id.bic"/>
+                    <CodicePagamento t-esc="format_alphanumeric(record.payment_reference[:60]) or '.'"/>
```

Comment:
Ah, I have to remove this `or '.'`, it was needed in a test... but I have to fix the test instead